README.md

```diff
@@ -1,8 +1,22 @@
 # libres [![Libres testing](https://github.com/equinor/libres/workflows/Libres%20testing/badge.svg)](https://github.com/equinor/libres/actions?query=workflow%3A%22Libres+testing%22) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-`libres` is part of the `ERT` project: _[Ensemble based Reservoir Tool](https://github.com/Equinor/ert)_.
+*libres* is part of the `ERT` project: _[Ensemble based Reservoir Tool](https://github.com/Equinor/ert)_. It is now available in PyPI:
 
-## Building libres
+``` sh
+$ pip install equinor-libres
+```
+
+or, for the latest development version:
+
+``` sh
+$ pip install git+https://github.com/equinor/libres.git@master
+```
+
+## Building (Legacy)
+
+*libres* is meant to be installed using `setup.py`, rather than `CMakeLists.txt`.
+
+*libres* is a C library with Python bindings. The C library requires [*libecl*](https://github.com/equinor/libecl)
```

Comment:
I'm not sure if it matters, but our "official statement" the last year or so is that we're a Python library with (large) parts implemented in C.